Tests/MongoSwiftSyncTests/MongoCollection+IndexTests.swift

```diff
@@ -285,6 +285,61 @@ final class MongoCollection_IndexTests: MongoSwiftTestCase {
         expect(receivedEvents[1].command["maxTimeMS"]).toNot(beNil())
         expect(receivedEvents[1].command["maxTimeMS"]?.toInt()).to(equal(maxTimeMS))
     }
+
+    func testCreateListDropIndexesComment() throws {
+        let comment = BSON("hello world")
+        let maxTimeMS = 5000
```

Comment:
since we're only testing comment here, we can omit this and the write concern in the options below.